app/src/main/java/org/mozilla/fenix/library/history/HistoryFragmentStore.kt

```diff
@@ -30,6 +30,8 @@ sealed class HistoryFragmentAction : Action {
     object ExitEditMode : HistoryFragmentAction()
     data class AddItemForRemoval(val item: HistoryItem) : HistoryFragmentAction()
     data class RemoveItemForRemoval(val item: HistoryItem) : HistoryFragmentAction()
+    data class AddPendingDeletionSet(val itemIds: Set<Long>) : HistoryFragmentAction()
+    data class UndoRemovePendingDeletionSet(val itemIds: Set<Long>) : HistoryFragmentAction()
```

Comment:
Let's call this `UndoPendingDeletionSet`

Reply:
Agreed, thanks for catching that.